sites/fast-website/src/app/css/style.css

```diff
@@ -1,68 +1,79 @@
-body {
-  margin: 0;
-  padding: 0;
-}
-
-.wrapper {
-  display: grid;
-  grid-template-columns: minmax(5vw, 1fr) minmax(0px, 1600px) minmax(5vw, 1fr);
-}
-
-.header {
-  grid-column: 2;
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  height: 80px;
-}
-
-.section {
-  grid-column: 2;
-}
-
-.section-content {
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  min-height: 300px;
-  margin: 20px 0;
-}
-
-.section-component {
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  height: 600px;
-  background-color: #3A3A3A;
-}
-
-.footer {
-  grid-column: 2;
-  height: 240px;
-  margin-top: 100px
-}
-
-.footer-links {
-  display: flex;
-  justify-content: space-between;
-}
-
-.footer-copyright-group {
-  display: flex;
-}
-
-.section-badge {
-  font-weight: bold;
-}
-
-.section-heading-hero {
-  font-size: 60px;
-  line-height: 72px;
-  margin: 0;
-}
-
-.section-heading {
-  font-size: 46px;
-  line-height: 56px;
-  margin: 0;
-}
\ No newline at end of file
+:root {
+  /* FIXME: Is there a better way to set default props to prevent FOUC? */
```

Comment:
Try making the above provider reference change and see if this still repros.

Reply:
After making the change above, I'm still getting the same white flash without setting at least this var.

Reply:
hmm. Could be due to script loading after the DOM is parsed - I can look into this one